packages/design-system/src/DatetimePicker.stories.tsx

```
@@ -0,0 +1,139 @@
+      value={date}
+      defaultPopupValue={date}
+      onChange={setDate}
+      className={{ trigger: 'w-[280px]' }}
+      label="Testlabel"
+      labelType="small"
+      granularity="minute"
+    />
+  )
+}
+
+export function Error() {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)